processed 10 tasks

task 2 'run'. lines 53-66:
Error: Script execution failed with VMError: {
    message: Expected type u64, got u32 which is not assignable  @Unknown invariant violation generated:
In function backtrace::backtrace::libunwind::trace::h619dd0d3ad107026 at /Users/wrwg/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/backtrace-0.3.69/src/backtrace/libunwind.rs:93
In function backtrace::backtrace::trace_unsynchronized::hf3cbad3825606d22 at /Users/wrwg/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/backtrace-0.3.69/src/backtrace/mod.rs:66
In function backtrace::backtrace::trace::h18bb3e7df594df3f at /Users/wrwg/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/backtrace-0.3.69/src/backtrace/mod.rs:53
In function move_binary_format::errors::PartialVMError::new::hf8844d34d2949273 at /Users/wrwg/aptos-core/third_party/move/move-binary-format/src/errors.rs:422
In function move_vm_types::loaded_data::runtime_types::Type::paranoid_check_assignable::h76efc649f2f6b549 at /Users/wrwg/aptos-core/third_party/move/move-vm/types/src/loaded_data/runtime_types.rs:382
In function move_vm_runtime::runtime_type_checks::verify_pack_closure::{{closure}}::h5316c6aef20d00cb at /Users/wrwg/aptos-core/third_party/move/move-vm/runtime/src/runtime_type_checks.rs:163


State: >>>>>>>>>>>>
Call stack:
*frame #0: script::main [pc = 4]:
0> LdU64(1)
1> StLoc(0)
2> ImmBorrowLoc(0)
3> LdU32(2)
4* PackClosure(0, 1)
Locals:
[0] U64(1)
Operand Stack:
(&container)[0]
0x0000000000000000000000000000000000000000000000000000000000000066::test::f(U32(2), ..)

<<<<<<<<<<<<
,
    major_status: UNKNOWN_INVARIANT_VIOLATION_ERROR,
    sub_status: Some(1),
    location: script,
    indices: [],
    offsets: [(FunctionDefinitionIndex(0), 4)],
    exec_state: Some(ExecutionState { stack_trace: [] }),
}

task 3 'run'. lines 68-81:
Error: Script execution failed with VMError: {
    message: Expected type u64, got u32 which is not assignable  @Unknown invariant violation generated:
In function backtrace::backtrace::libunwind::trace::h619dd0d3ad107026 at /Users/wrwg/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/backtrace-0.3.69/src/backtrace/libunwind.rs:93
In function backtrace::backtrace::trace_unsynchronized::hf3cbad3825606d22 at /Users/wrwg/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/backtrace-0.3.69/src/backtrace/mod.rs:66
In function backtrace::backtrace::trace::h18bb3e7df594df3f at /Users/wrwg/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/backtrace-0.3.69/src/backtrace/mod.rs:53
In function move_binary_format::errors::PartialVMError::new::hf8844d34d2949273 at /Users/wrwg/aptos-core/third_party/move/move-binary-format/src/errors.rs:422
In function move_vm_types::loaded_data::runtime_types::Type::paranoid_check_assignable::h76efc649f2f6b549 at /Users/wrwg/aptos-core/third_party/move/move-vm/types/src/loaded_data/runtime_types.rs:382
In function move_vm_types::loaded_data::runtime_types::Type::paranoid_check_assignable::h76efc649f2f6b549 at /Users/wrwg/aptos-core/third_party/move/move-vm/types/src/loaded_data/runtime_types.rs:517


State: >>>>>>>>>>>>
Call stack:
*frame #0: script::main [pc = 5]:
0> LdU32(1)
1> StLoc(0)
2> ImmBorrowLoc(0)
3> LdU64(2)
4> PackClosure(0, 1)
5* CallClosure(3)
Locals:
[0] U32(1)
Operand Stack:

<<<<<<<<<<<<
,
    major_status: UNKNOWN_INVARIANT_VIOLATION_ERROR,
    sub_status: Some(1),
    location: undefined,
    indices: [],
    offsets: [],
    exec_state: None,
}

task 4 'run'. lines 83-98:
Error: Script execution failed with VMError: {
    message: Expected type |u64|u64, got |&u64|u64 has copy + drop + store which is not assignable  @Unknown invariant violation generated:
In function backtrace::backtrace::libunwind::trace::h619dd0d3ad107026 at /Users/wrwg/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/backtrace-0.3.69/src/backtrace/libunwind.rs:93
In function backtrace::backtrace::trace_unsynchronized::hf3cbad3825606d22 at /Users/wrwg/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/backtrace-0.3.69/src/backtrace/mod.rs:66
In function backtrace::backtrace::trace::h18bb3e7df594df3f at /Users/wrwg/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/backtrace-0.3.69/src/backtrace/mod.rs:53
In function move_binary_format::errors::PartialVMError::new::hf8844d34d2949273 at /Users/wrwg/aptos-core/third_party/move/move-binary-format/src/errors.rs:422
In function move_vm_types::loaded_data::runtime_types::Type::paranoid_check_assignable::h76efc649f2f6b549 at /Users/wrwg/aptos-core/third_party/move/move-vm/types/src/loaded_data/runtime_types.rs:382
In function <move_vm_runtime::runtime_type_checks::FullRuntimeTypeCheck as move_vm_runtime::runtime_type_checks::RuntimeTypeCheck>::pre_execution_type_stack_transition::ha05aeb35c28606cd at /Users/wrwg/aptos-core/third_party/move/move-vm/runtime/src/runtime_type_checks.rs:273


State: >>>>>>>>>>>>
Call stack:
*frame #0: script::main [pc = 5]:
0> LdU64(1)
1> StLoc(0)
2> ImmBorrowLoc(0)
3> LdU64(2)
4> PackClosure(0, 1)
5* CallClosure(3)
Locals:
[0] U64(1)
Operand Stack:
(&container)[0]
0x0000000000000000000000000000000000000000000000000000000000000066::test::f(U64(2), ..)

<<<<<<<<<<<<
,
    major_status: UNKNOWN_INVARIANT_VIOLATION_ERROR,
    sub_status: Some(1),
    location: script,
    indices: [],
    offsets: [(FunctionDefinitionIndex(0), 5)],
    exec_state: Some(ExecutionState { stack_trace: [] }),
}

task 6 'run'. lines 115-128:
Error: Script execution failed with VMError: {
    message: Expected type u64, got u32 which is not assignable  @Unknown invariant violation generated:
In function backtrace::backtrace::libunwind::trace::h619dd0d3ad107026 at /Users/wrwg/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/backtrace-0.3.69/src/backtrace/libunwind.rs:93
In function backtrace::backtrace::trace_unsynchronized::hf3cbad3825606d22 at /Users/wrwg/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/backtrace-0.3.69/src/backtrace/mod.rs:66
In function backtrace::backtrace::trace::h18bb3e7df594df3f at /Users/wrwg/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/backtrace-0.3.69/src/backtrace/mod.rs:53
In function move_binary_format::errors::PartialVMError::new::hf8844d34d2949273 at /Users/wrwg/aptos-core/third_party/move/move-binary-format/src/errors.rs:422
In function move_vm_types::loaded_data::runtime_types::Type::paranoid_check_assignable::h76efc649f2f6b549 at /Users/wrwg/aptos-core/third_party/move/move-vm/types/src/loaded_data/runtime_types.rs:382
In function move_vm_runtime::runtime_type_checks::verify_pack_closure::{{closure}}::h5316c6aef20d00cb at /Users/wrwg/aptos-core/third_party/move/move-vm/runtime/src/runtime_type_checks.rs:163


State: >>>>>>>>>>>>
Call stack:
*frame #0: script::main [pc = 4]:
0> LdU64(1)
1> StLoc(0)
2> ImmBorrowLoc(0)
3> LdU32(2)
4* PackClosure(0, 10)
Locals:
[0] U64(1)
Operand Stack:
(&container)[0]
0x0000000000000000000000000000000000000000000000000000000000000066::test::g(_, U32(2), ..)

<<<<<<<<<<<<
,
    major_status: UNKNOWN_INVARIANT_VIOLATION_ERROR,
    sub_status: Some(1),
    location: script,
    indices: [],
    offsets: [(FunctionDefinitionIndex(0), 4)],
    exec_state: Some(ExecutionState { stack_trace: [] }),
}

task 7 'run'. lines 130-147:
Error: Script execution failed with VMError: {
    message: Expected type u64, got u32 which is not assignable  @Unknown invariant violation generated:
In function backtrace::backtrace::libunwind::trace::h619dd0d3ad107026 at /Users/wrwg/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/backtrace-0.3.69/src/backtrace/libunwind.rs:93
In function backtrace::backtrace::trace_unsynchronized::hf3cbad3825606d22 at /Users/wrwg/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/backtrace-0.3.69/src/backtrace/mod.rs:66
In function backtrace::backtrace::trace::h18bb3e7df594df3f at /Users/wrwg/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/backtrace-0.3.69/src/backtrace/mod.rs:53
In function move_binary_format::errors::PartialVMError::new::hf8844d34d2949273 at /Users/wrwg/aptos-core/third_party/move/move-binary-format/src/errors.rs:422
In function move_vm_types::loaded_data::runtime_types::Type::paranoid_check_assignable::h76efc649f2f6b549 at /Users/wrwg/aptos-core/third_party/move/move-vm/types/src/loaded_data/runtime_types.rs:382
In function move_vm_types::loaded_data::runtime_types::Type::paranoid_check_assignable::h76efc649f2f6b549 at /Users/wrwg/aptos-core/third_party/move/move-vm/types/src/loaded_data/runtime_types.rs:517


State: >>>>>>>>>>>>
Call stack:
*frame #0: script::main [pc = 5]:
0> LdU32(1)
1> StLoc(0)
2> ImmBorrowLoc(0)
3> LdU64(2)
4> PackClosure(0, 10)
5* CallClosure(3)
Locals:
[0] U32(1)
Operand Stack:

<<<<<<<<<<<<
,
    major_status: UNKNOWN_INVARIANT_VIOLATION_ERROR,
    sub_status: Some(1),
    location: undefined,
    indices: [],
    offsets: [],
    exec_state: None,
}

task 8 'run'. lines 149-162:
Error: Script execution failed with VMError: {
    message: No type layout for Reference(U64) @Unknown invariant violation generated:
In function backtrace::backtrace::libunwind::trace::h619dd0d3ad107026 at /Users/wrwg/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/backtrace-0.3.69/src/backtrace/libunwind.rs:93
In function backtrace::backtrace::trace_unsynchronized::hf3cbad3825606d22 at /Users/wrwg/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/backtrace-0.3.69/src/backtrace/mod.rs:66
In function backtrace::backtrace::trace::h18bb3e7df594df3f at /Users/wrwg/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/backtrace-0.3.69/src/backtrace/mod.rs:53
In function move_binary_format::errors::PartialVMError::new::hf8844d34d2949273 at /Users/wrwg/aptos-core/third_party/move/move-binary-format/src/errors.rs:422
In function move_vm_runtime::storage::ty_layout_converter::LayoutConverterBase::type_to_type_layout_impl::h317d7bdc3e788316 at /Users/wrwg/aptos-core/third_party/move/move-vm/runtime/src/storage/ty_layout_converter.rs:157
In function move_vm_runtime::storage::ty_layout_converter::LayoutConverter::type_to_type_layout_with_identifier_mappings::h57d88b7a2fe5e4e6 at /Users/wrwg/aptos-core/third_party/move/move-vm/runtime/src/storage/ty_layout_converter.rs:38


State: >>>>>>>>>>>>
Call stack:
*frame #0: script::main [pc = 3]:
0> LdU64(1)
1> StLoc(0)
2> ImmBorrowLoc(0)
3* PackClosure(0, 1)
Locals:
[0] U64(1)
Operand Stack:

<<<<<<<<<<<<
,
    major_status: UNKNOWN_INVARIANT_VIOLATION_ERROR,
    sub_status: None,
    location: script,
    indices: [],
    offsets: [(FunctionDefinitionIndex(0), 3)],
    exec_state: Some(ExecutionState { stack_trace: [] }),
}

task 9 'run'. lines 164-177:
Error: Script execution failed with VMError: {
    message: No type layout for MutableReference(U64) @Unknown invariant violation generated:
In function backtrace::backtrace::libunwind::trace::h619dd0d3ad107026 at /Users/wrwg/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/backtrace-0.3.69/src/backtrace/libunwind.rs:93
In function backtrace::backtrace::trace_unsynchronized::hf3cbad3825606d22 at /Users/wrwg/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/backtrace-0.3.69/src/backtrace/mod.rs:66
In function backtrace::backtrace::trace::h18bb3e7df594df3f at /Users/wrwg/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/backtrace-0.3.69/src/backtrace/mod.rs:53
In function move_binary_format::errors::PartialVMError::new::hf8844d34d2949273 at /Users/wrwg/aptos-core/third_party/move/move-binary-format/src/errors.rs:422
In function move_vm_runtime::storage::ty_layout_converter::LayoutConverterBase::type_to_type_layout_impl::h317d7bdc3e788316 at /Users/wrwg/aptos-core/third_party/move/move-vm/runtime/src/storage/ty_layout_converter.rs:157
In function move_vm_runtime::storage::ty_layout_converter::LayoutConverter::type_to_type_layout_with_identifier_mappings::h57d88b7a2fe5e4e6 at /Users/wrwg/aptos-core/third_party/move/move-vm/runtime/src/storage/ty_layout_converter.rs:38


State: >>>>>>>>>>>>
Call stack:
*frame #0: script::main [pc = 3]:
0> LdU64(1)
1> StLoc(0)
2> MutBorrowLoc(0)
3* PackClosure(0, 1)
Locals:
[0] U64(1)
Operand Stack:

<<<<<<<<<<<<
,
    major_status: UNKNOWN_INVARIANT_VIOLATION_ERROR,
    sub_status: None,
    location: script,
    indices: [],
    offsets: [(FunctionDefinitionIndex(0), 3)],
    exec_state: Some(ExecutionState { stack_trace: [] }),
}
